_goldens/test/_files/directives/directive_wrapper.template.golden

```diff
@@ -67,10 +67,6 @@ class ViewTestFooComponent0 extends AppView<import1.TestFooComponent> {
     }
     _ChildDirective_0_5.detectHostChanges(this, _el_0);
   }
-
-  void _handle_trigger_0_0($event) {
-    ctx.onTrigger;
```

Comment:
This just seems...weird. Did this ever do anything? Nothing caller or returned?

Reply:
No, I believe this golden was just put in place so we could see the change.